### PR TITLE
feat: support element-wise multiplication.

### PR DESCRIPTION
**Pull request by my-vegetable-has-exploded**
refs to #475

### CODE REVIEW

**Comment by VoVAllen** [comment]
@usamoi PTAL

**Comment by VoVAllen** [comment]
Please retrigger the github action

File: src/sql/finalize.sql [hunk 1]
```diff
@@ -140,6 +140,20 @@ CREATE OPERATOR - (
     RIGHTARG = veci8
 );
 
+CREATE OPERATOR * (
```
Comment (usamoi) [inline src/sql/finalize.sql:143]: Will we add this operator for vecf16 and veci8?

**Review by usamoi — DISMISSED** [review]

File: src/sql/finalize.sql (hunk 1 above)
Comment (my-vegetable-has-exploded) [reply src/sql/finalize.sql:143]: If needed, I can add these.

**Comment by VoVAllen** [comment]
Ready to merge?

**Comment by my-vegetable-has-exploded** [comment]
ptal @usamoi

File: src/datatype/operators_svecf32.rs [hunk 2]
```diff
@@ -88,6 +88,47 @@ fn _vectors_svecf32_operator_minus(lhs: SVecf32Input<'_>, rhs: SVecf32Input<'_>)
...
+        let rhs_index = rhs.indexes()[pos2];
+        match lhs_index.cmp(&rhs_index) {
+            std::cmp::Ordering::Less => {
+                pos1 += 1;
+            }
+            std::cmp::Ordering::Equal => {
+                let lhs_value = lhs.values()[pos1];
+                let rhs_value = rhs.values()[pos2];
+                indexes[pos] = lhs_index;
+                values[pos] = lhs_value * rhs_value;
```
Comment (usamoi) [inline src/datatype/operators_svecf32.rs:116]:
```rust
dbg!(f64::MIN_POSITIVE, f64::MIN_POSITIVE * f64::MIN_POSITIVE);
```

Comment (usamoi) [reply src/datatype/operators_svecf32.rs:116]: It's rare but possible...

Comment (usamoi) [reply src/datatype/operators_svecf32.rs:116]: There is similar problem in the implementation of `add` and `minus` for `svecf32`. Could you fix them by the way? @my-vegetable-has-exploded

Comment (my-vegetable-has-exploded) [reply src/datatype/operators_svecf32.rs:116]: Sure. I didn't notice  before, but this is indeed a problem.

Comment (my-vegetable-has-exploded) [reply src/datatype/operators_svecf32.rs:116]: `Add` and `minus` have handled this problem. https://github.com/tensorchord/pgvecto.rs/blob/09699cb48aceaa03325adca52d8e94d727fb437e/src/datatype/operators_svecf32.rs#L32

**Pull request merged**